legal-api/src/legal_api/services/authz.py

```diff
@@ -153,7 +154,10 @@ def has_roles(jwt: JwtManager, roles: List[str]) -> bool:
                 }
             },
             'conversion': {
-                'legalTypes': ['SP', 'GP']
+                'legalTypes': ['SP', 'GP'],
+                'blockerChecks': {
+                    'business': [BusinessBlocker.ADMN_FRZE]
+                }
```

Comment:
from Mihai's latest comments, frozen business should not block the conversion filing.  i.e. revert to original code where there are no blocker checks for the conversion filing.

Reply:
Sorry, I missed it. I updated.